### PR TITLE
Add `BitSplitButton` component (#2972)

### LINKED ISSUE #2972: Add `BitSplitButton` component
based on [Fluent](https://developer.microsoft.com/en-us/fluentui#/controls/web/button) we need a new component in the name of `SplitButton`.

Untitled

### PR DESCRIPTION
**Pull request by sajadkardel**
this closes #2972

### CODE REVIEW

**Comment by sajadkardel** [comment]
https://user-images.githubusercontent.com/62190682/185267334-5c3ef6cb-9957-42e3-8b63-41df17593c22.MOV

File: src/BlazorUI/Bit.BlazorUI/Components/Buttons/BitSplitButton/BitSplitButton.Fluent.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿@import "../../../Styles/Fluent/colors.fluent.scss", "../../../Styles/functions.scss";
 
-.bit-splt-btn {
-    &.bit-splt-btn-fluent {
+.bit-spl {
```
Comment (msynk) [inline src/BlazorUI/Bit.BlazorUI/Components/Buttons/BitSplitButton/BitSplitButton.Fluent.scss:3]: it should be `spb` not `spl`

**Review by msynk — CHANGES_REQUESTED** [review]

File: src/BlazorUI/Bit.BlazorUI/Components/Buttons/BitSplitButton/BitSplitButton.Fluent.scss (hunk 1 above)
Comment (sajadkardel) [reply src/BlazorUI/Bit.BlazorUI/Components/Buttons/BitSplitButton/BitSplitButton.Fluent.scss:3]: The abbreviation `spb` has already been used in the `BitSpinButton` component and we cannot reuse it.

File: src/BlazorUI/Bit.BlazorUI/Components/Buttons/BitSplitButton/BitSplitButton.Fluent.scss [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+            }
+        }
+
+        &:hover {
+            background-color: $Grey20;
+        }
+    }
+}
+
+@keyframes opacity {
```
Comment (msynk) [inline src/BlazorUI/Bit.BlazorUI/Components/Buttons/BitSplitButton/BitSplitButton.Fluent.scss:219]: plz choose a better and more specific names for these keyframes elements

**Review by msynk — CHANGES_REQUESTED** [review]

**Review by msynk — APPROVED** [review]

**Pull request merged**